Google.PowerShell.IntegrationTests/Container2/Container.GkeNodePool.Tests.ps1

```diff
@@ -30,6 +30,31 @@ $clusterThreeCreationJob = Start-Job -ScriptBlock $clusterCreationScriptBlock `
 
 Wait-Job $clusterOneCreationJob, $clusterTwoCreationJob, $clusterThreeCreationJob | Remove-Job
 
+# Check that a cluster is running.
+function Check-Cluster($clusterName, $clusterZone) {
+    $cluster = Get-GkeCluster -ClusterName $clusterName -Zone $clusterZone
+    return $cluster.Status -eq "Running"
+}
+
+$clusterNames = @($clusterOneName, $clusterTwoName, $clusterThreeName)
+$clusterZones = @($zone, $clusterTwoZone, $clusterThreeZone)
+
+# Wait until every cluster is fully started. Wait for at most 10 minutes.
+$counter = 0
+for ($i = 0; $i -lt $clusterNames.Count; $i += 1) {
+    while ($true) {
+        if (-not (Check-Cluster $clusterNames[$i] $clusterZones[$i])) {
+            Start-Sleep -Seconds 30
+            $counter += 1
+            if ($counter -eq 20) {
+                break
+            }
+        }
+        else {
+            break
+        }
+    }
```

Comment:
The cluster is in provisioning state so we are just waiting for it to get into the running state.